sessions/backend-update-delete/demo-start/pages/api/jokes/[id].js

```diff
@@ -0,0 +1,17 @@
+import dbConnect from "../../../db/connect";
+import Joke from "../../../db/models/Joke";
+
+export default async function handler(request, response) {
+  await dbConnect();
+  const { id } = request.query;
+
+  if (request.method === "GET") {
```

Comment:
if the request method is not "GET" here, you will get a timeout, the other cases should be handled as well with a send of a status 405.

Reply:
Nice catch! :) For this, too, an issue already exists. (Totally understandable that you're unaware.) We'll try to get to this asap!